### PR TITLE
Created sawan12345.md

### PR DESCRIPTION
**Pull request by Sawan12345**
## test file
#18 Closed. Added bio for sawan12345.

### CODE REVIEW

File: sawan12345.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Student
```
Comment (typicalaimster) [inline sawan12345.md:1]: @Sawan12345 can you add detail kthx?

File: sawan12345.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+#Name : Sawan
+#Location : Bedford, MA
+#
```
Comment (kpineiro) [inline sawan12345.md]: @Sawan12345 Add favorite food please

File: sawan12345.md (hunk 1 above)
Comment (Sawan12345) [inline sawan12345.md:1]: @typicalaimster : added more detail.

File: sawan12345.md (hunk 2 above)
Comment (Sawan12345) [inline sawan12345.md]: @kpineiro  Samosha :)

**Comment by mclassen** [comment]
@Sawan12345: Please assign someone to this pull request.

**Comment by Sawan12345** [comment]
@mclassen : assigned to you.

**Comment by mclassen** [comment]
@githubteacher: can any contributor merge pull requests?

**Comment by Sawan12345** [comment]
Not sure.

**Comment by Sawan12345** [comment]
oh I can see that you have already merged it . Should i close that branch ?

**Pull request merged**